qfieldsync/ui/map_layer_config_widget.ui

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>QFieldLayerSettingsPage</class>
+ <widget class="QWidget" name="QFieldLayerSettingsPage">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>500</width>
+    <height>250</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Form</string>
+  </property>
+  <layout class="QFormLayout" name="QFieldLayerSettingsPageLayout">
+   <item row="0" column="0">
+    <widget class="QLabel" name="layerActionLabel">
+     <property name="text">
+      <string>Layer Action</string>
+     </property>
+    </widget>
+   </item>
+   <item row="0" column="1">
+    <widget class="QComboBox" name="layerActionComboBox">
+     <property name="sizePolicy">
+      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
+       <horstretch>0</horstretch>
+       <verstretch>0</verstretch>
+      </sizepolicy>
+     </property>
+    </widget>
+   </item>
+   <item row="1" column="1">
+    <widget class="QCheckBox" name="isGeometryLockedCheckBox">
+     <property name="toolTip">
+      <string>When enabled, this option disables adding and deleting features, as well as modifying the geometries of existing features.</string>
+     </property>
+     <property name="text">
+      <string>lock geometry</string>
```

Comment:
```suggestion
      <string>Lock Geometries</string>
```

Just the usual nitpicking

Reply:
NOTE that not all of the checkboxes are in This Style, e.g. `Create base map`

Reply:
https://docs.qgis.org/3.10/en/docs/developers_guide/hig.html

For what I can see, rule 3. applies here.

Reply:
Then as far as I understand it, it should be `Create Base Map`, we have a selectable widget `QCheckBox` and there are no prepositions, conjuctions and articles that are the only exceptions? 

I don't know, I just checked other QGIS interfaces, it seems I am not the only one who got confused by rule 3 :)

Reply:
Yes, that's what it should be. QGIS has organically grown and it's a movement towards Zen

Reply:
I will make a separate PR on this later, there are few more captions that don't follow the rule